README.md

```diff
@@ -14,30 +14,55 @@ Apollo. You may find references to these previous names in code or
 documentation.
 
 ## Community
-You can reach out to us through [Slack](https://cloud-native.slack.com/archives/C01TDE3GK0E) (#stackrox).  
-For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackrox.io/).
+You can chat directly with us on the [#stackrox channel in the CNCF Slack](https://www.stackrox.io/slack/).
+
+For event updates, blogs and other resources follow the StackRox community site at [stackrox.io](https://www.stackrox.io/).
 
 ## Table of contents
 
-* [Quick Installation via Helm](#quick-installation-via-helm)
-    + [First use](#first-use)
-* [Manual Deployment](#manual-deployment)
-* [Development](#development)
-    + [Quickstart](#quickstart)
+- [StackRox Kubernetes Security Platform](#stackrox-kubernetes-security-platform)
+  - [Community](#community)
+  - [Table of contents](#table-of-contents)
+  - [Deploying StackRox](#deploying-stackrox)
+    - [Installation via Helm](#installation-via-helm)
+      - [Default Installation](#default-installation)
+      - [Limited Resource Installation](#limited-resource-installation)
+      - [Default Installation](#default-installation-1)
+      - [Limited Resource Installation](#limited-resource-installation-1)
```

Comment:
Same thing here:
The rendered ToC really looks a bit weird, as we have "Default Installation" and "Limited Reource Installation" two times. Should we maybe change the headings of the second pair of these to something else? 
Because right now it looks like a bug. Especially as none of the four hyperlinks render correctly in GitHub (at least if you do a rendered preview of this Markdown).

Reply:
I updated it to provide even more clarity.